src/sentry/features/handler.py

```diff
@@ -17,6 +17,13 @@
 
 
 class FeatureHandler:
+    """
+    Base class for defining custom logic for feature decisions.
+
+    Subclasses should implement `has` and contain the logic
+    necessary for the feature check.
+    """
+
```

Comment:
Since this is the root it may be helpful to enumerate common use-cases, any recommended do's and don'ts, file locations to place flags, links to enabling flags in relay and other services, etc.

Reply:
I've added a bit more context, and can go through the getsentry code and expand documentation there as well.